docs/media.html

```diff
@@ -93,10 +88,6 @@ <h4 class="">Media Three</h4>
       integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
```

Comment:
I'm noticing that you left a copy of jQuery here, but it's not necessary with Bootstrap 5

Reply:
Got it. I'll remember to remove that everywhere.